src/core/InputField/styles.ts

```diff
@@ -18,6 +18,30 @@ const useStyles = makeStyles((theme: Theme) => ({
     '& .MuiSvgIcon-root': {
       color: theme.palette.text.hint,
     },
+    '& .MuiInputBase-input': {
+      '&:focus': {
+        '&:-webkit-autofill': {
+          WebkitBoxShadow: `0 0 0 1000px ${theme.palette.background.paper} inset`,
+          WebkitTextFillColor: theme.palette.text.primary,
+        },
+      },
+      '&:hover': {
+        '&:-webkit-autofill': {
+          WebkitBoxShadow: `0 0 0 1000px ${theme.palette.background.paper} inset`,
+          WebkitTextFillColor: theme.palette.text.primary,
+        },
+      },
+      '&:active': {
+        '&:-webkit-autofill': {
+          WebkitBoxShadow: `0 0 0 1000px ${theme.palette.background.paper} inset`,
+          WebkitTextFillColor: theme.palette.text.primary,
+        },
+      },
+      '&:-webkit-autofill': {
+        WebkitBoxShadow: `0 0 0 1000px ${theme.palette.background.paper} inset`,
+        WebkitTextFillColor: theme.palette.text.primary,
+      },
```

Comment:
Since these are all sharing the same property, I guess you can write them as
```suggestion
      '&:focus',
      '&:hover',
      '&:active': {
        '&:-webkit-autofill': {
          WebkitBoxShadow: `0 0 0 1000px ${theme.palette.background.paper} inset`,
          WebkitTextFillColor: theme.palette.text.primary,
        },
      },
      '&:-webkit-autofill': {
        WebkitBoxShadow: `0 0 0 1000px ${theme.palette.background.paper} inset`,
        WebkitTextFillColor: theme.palette.text.primary,
      },
```

Reply:
I was keeping them separate incase other colors were to be used. Will make the changes.